src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -40,6 +40,11 @@ AppHost::AppHost() noexcept :
                          std::placeholders::_3);
     _window->SetCreateCallback(pfn);
 
+    _window->SetSnapDimensionCallback(std::bind(&winrt::TerminalApp::AppLogic::CalcSnappedDimension,

```

Comment:
I believe @DHowett-MSFT has a way to avoid `std::bind` for things like this and the full reasoning why (which has something to do with stack frames, I believe). I think it's Lambda related. My brain is just full of the holiday fuzzies on the details.

Reply:
That's #3181

Reply:
OK so I really need to go read #3181. Apologies that I didn't do that first. It's on my list for reading today.